test/sr_sessions_SUITE.erl

```diff
@@ -273,7 +274,67 @@ location(Config) ->
 
   {comment, ""}.
 
+-spec test_coverage(sr_test_utils:config()) -> {comment, string()}.
+test_coverage(Config) ->
+  {basic_auth, BasicAuth} = lists:keyfind(basic_auth, 1, Config),
+  Headers = #{ basic_auth => BasicAuth
+             , <<"content-type">> => <<"application/json">>
+             },
+
+  ct:comment("Create a session with put and thru sr_single_session_handler"),
+  ok = meck:expect(sr_single_session_handler, is_conflict, fun(Req, State) ->
+    {false, Req, State}
+  end),
+  #{status_code := 201, body := _Body} =
+    sr_test_utils:api_call( put
+                          , "/sessions/this_is_an_ID"
+                          , Headers
+                          , #{agent => <<"a1">>}),
+  [_] = meck:unload(),
+
+  ct:comment("Functions coverage"),
+  SrState = sr_state:new([], fake_module_name),
+  undefined = sr_state:id(SrState),
+  SrState2 = sr_state:set(key, value, SrState),
+  value = sr_state:retrieve(key, SrState2, not_found),
+  SrState3 = sr_state:remove(key, SrState2),
+  not_found = sr_state:retrieve(key, SrState3, not_found),
+  CowboyReq = fake_cowboy_req(),
```

Comment:
Can we use the functions in `sr_request` in a super-dumb echo handler/module with the following function?

```erlang
from_ctx(#{req := Req}) ->
  #{headers => sr_request:headers(Req),
    path => sr_request:path(Req),
    bindings => sr_request:bindings(Req)
  }.
```

My goal is to test these new features by actually using them, not just for coverage completeness